doc/api/Creating_a_Player.md

```diff
@@ -104,11 +104,6 @@ The internal checks of the RxPlayer is based on an estimation of what the RxPlay
 is currently buffered and an estimation of the size of the next segments.
 </div>
 
-<div class="warning">
```

Comment:
Similar to the warning below

Reply:
It was to make sure the warning was taken into account.

jk, probably appeared after some bad conflict resolution.